### DISCUSSION TITLE
Error: Unable to establish a connection to query-engine-node-api library. It seems there is a problem with your OpenSSL installation! Details: Unable to require(`/usr/src/app/node_modules/prisma/libquery_engine-linux-musl.so.node`)  Error loading shared library libssl.so.1.1: No such file or directory (needed by /usr/src/app/node_modules/prisma/libquery_engine-linux-musl.so.node)

### DISCUSSION THREAD

**Discussion by ansible-rdonahue** [prisma/prisma#16789] [Q&A] [answered]
Hello, I am running a job to build, tag, and push to AWS ECR., with the following error:
```
Error: Unable to establish a connection to query-engine-node-api library. It seems there is a problem with your OpenSSL installation!
Details: Unable to require(`/usr/src/app/node_modules/prisma/libquery_engine-linux-musl.so.node`)
 Error loading shared library libssl.so.1.1: No such file or directory (needed by /usr/src/app/node_modules/prisma/libquery_engine-linux-musl.so.node)
```

I'm using Prisma CLI Version : 4.3.1

When running where `openssl` it returns `/usr/bin/openssl`, noting because some similar issues folx reported showed multiple installations and one was overriding another.  

Any suggestions on how to resolve?

**Comment by ansible-rdonahue** [comment, marked as answer]
We reran the job and the error went away, with no changes.  Not sure what caused it but wanted to give an update that the issue is no longer happening.

**Reply by nurul3101** [reply]
Glad to hear that the issue is resolved, if it is a persistent issue, please feel free to create a new GitHub Discussion with details so that we can take a look.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
